linux/ibus-keyman/.gitignore

```diff
@@ -6,6 +6,7 @@ src/ibus-engine-keyman
 src/print_kmp
 src/print_kmpdetails
 src/test/keymanutil_tests
+src/test/keymanutil-tests
```

Comment:
Do we have both of these folders? That seems a bit confusing!

Reply:
Note that these are the test executables. I thought that I'd create both, depending on how we build, but it seems that was a left-over.